### ISSUE TITLE
Temporarily disable configuration cache

### ISSUE THREAD

**Issue by mvicsokolova** [Kotlin/kotlinx-atomicfu#483]
With enabled configuration cache, `atomicfu:publish` started to fail with the following error: 
```
Invocation of 'Task.project' by task ':atomicfu:cinteropInteropAndroidNativeArm32' at execution time is unsupported.
Invocation of 'Task.project' by task ':atomicfu:cinteropTestInteropAndroidNativeArm32' at execution time is unsupported.
Invocation of 'Task.project' by task ':atomicfu:cinteropInteropAndroidNativeArm64' at execution time is unsupported.
Invocation of 'Task.project' by task ':atomicfu:cinteropTestInteropAndroidNativeArm64' at execution time is unsupported.
Invocation of 'Task.project' by task ':atomicfu:cinteropInteropAndroidNativeX64' at execution time is unsupported.
```
 I could not reproduce this locally, but here is the [Windows deploy build](https://teamcity.jetbrains.com/buildConfiguration/KotlinTools_KotlinxAtomicfu_DeployWindows?branch=%3Cdefault%3E&buildTypeTab=overview&mode=builds&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildProblemsSection=true#4850331) and the [gradle scan](https://ge.jetbrains.com/s/w3t4jnffbe62m/failure?expanded-stacktrace=WyIwLTIiXQ#1) of this build. 

Seems (according to the scan), that the problem may be caused by the access to the `project` during execution from `org.jetbrains.kotlin.gradle.tasks.CInteropProcess#toGist()` which is incompatible with configuration caches and causes this problem. 

So, for now I'll disable configuration cache in the project, so that the release could be published.

**Comment by mvicsokolova** [comment]
@dkrasnoff could you please take a look at this problem 🙏
This only reproduced at deploy and was not discovered during review :(

**Comment by dkrasnoff** [comment]
This is a really nasty problem because, as I understand, `org.JetBrains.kotlin.gradle.tasks.CInteropProcess#toGist()` just hides the real problem.
I could not reproduce it locally as well, even on a Windows machine.

**Comment by dkrasnoff** [comment]
Yeah, as I thought the problem is not in the KGP and CInteropProcess, but they are hiding the real problem under false-positive error messages. 
The real problem is in credential block in publishing, which is [unsupported by Configuration Cache now](https://github.com/gradle/gradle/issues/24040)
